docs/data-cookbooks/setup.md

```diff
@@ -10,18 +10,96 @@ In the following data cookbooks we'll go through things like setting up workflow
 
 ## Schemas
 
-Looking at our api schema [definitions](https://github.com/nasa/cumulus/tree/master/packages/api/models/schemas.js) can provide us with some insight into collections, providers, rules, and their attributes (and whether those are required or not). The schema for different concepts will be reference throughout this document.
+Looking at our API schema [definitions](https://github.com/nasa/cumulus/tree/master/packages/api/models/schemas.js) can provide us with some insight into collections, providers, rules, and their attributes (and whether those are required or not). The schema for different concepts will be reference throughout this document.
```

Comment:
I have been thinking about moving this page (where?) but at least wanted to get a PR up with the diffs first.

Reply:
I was thinking I could move it into the data management types page, instead of linking to it.

Reply:
Definitely +1 to moving this. I think it's unfortunate that this information is hidden here

Reply:
Ok I'm going to open up a PR after this one is merged moving it so we maintain these changes in this PR.